base/features/spock/feature.yml

```diff
@@ -10,3 +10,5 @@ dependencies:
         module: groovy-all
   - scope: testCompile
     coords: io.micronaut:micronaut-inject-groovy
+  - scope: testCompile
+    coords: io.micronaut.test:micronaut-test-spock:1.0.1
```

Comment:
Should remove hard coded version here as it is already in the BOM https://github.com/micronaut-projects/micronaut-core/blob/master/build.gradle#L143